pkg/controllers/provisioningv2/cluster/controller.go

```diff
@@ -45,6 +45,7 @@ const (
 	administratedAnn      = "provisioning.cattle.io/administrated"
 	mgmtClusterNameAnn    = "provisioning.cattle.io/management-cluster-name"
 	fleetWorkspaceNameAnn = "provisioning.cattle.io/fleet-workspace-name"
+	turtlesOwnedLab       = "cluster-api.cattle.io/owned"
```

Comment:
```suggestion
	externallyManagedAnn       = "provisioning.cattle.io/externally-managed"
```

I think this is better suited as an annotation